pydantic/fields.py

```diff
@@ -101,7 +102,9 @@ class FieldInfo(_repr.Representation):
         frozen: Whether the field is frozen.
         validate_default: Whether to validate the default value of the field.
         repr: Whether to include the field in representation of the model.
-        init_var: Whether the field should be included in the constructor of the dataclass.
+        init: If true (the default), this field is included as a parameter to the generated `__init__` of the dataclass
+        init_var: Whether the field is an "init-only" field: used as a parameter to the generated `__init__` method, but
```

Comment:
From https://docs.pydantic.dev/latest/concepts/fields/#dataclass-constraints, and looking at a few of the test cases, i think the previous docstring for `init_var` needed to be tweaked.